packages/terra-consumer-nav/src/Layout.scss

```diff
@@ -24,9 +24,22 @@
     top: 0;
     width: calc(100vw - #{$terra-consumer--nav-container-width});
     @media screen and (max-width: $terra-medium-breakpoint) {
-      left: 0;
+      left: 20;
       max-width: 100vw;
       width: 100vw;
     }
   }
+  .footer {
+    width: 100%;
+    bottom: 15px;
+    margin:0;
+    padding:0;
+    position:sticky;
```

Comment:
Based on UX callout the help button is positioned in the bottom left corner when the page content takes full screen but it would be moved up if the content does not fill up the full screen. Here is the desktop UI callout.
https://wiki.ucern.com/pages/viewpage.action?pageId=1332770962&preview=/1332770962/1623759251/healf_ui-uplift_navigation_callouts_v2.pdf